sdks/rust/golem-rust/src/agentic/schema.rs

```diff
@@ -77,3 +78,28 @@ impl<T: IntoValue + FromValueAndType> Schema for T {
         }
     }
 }
+
+impl Schema for () {
+    fn get_type() -> ElementSchema {
+        let analysed_type = AnalysedType::Result(TypeResult {
```

Comment:
This type represents `Result<(), ()>` and not `()`. The schema for `()` should be the empty tuple data schema, same as we did in the TS SDK. There is no way to express `()` as a component model type